delfin/task_manager/scheduler/scheduler.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2021 The SODA Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from apscheduler.schedulers.background import BackgroundScheduler
+
+
+class Scheduler:
```

Comment:
If want to use BackgroundScheduler as singleton, delfin.utils.Singleton could be used for BackgroundScheduler

Reply:
Added singleton decorator.
Ref: sushanthakumar#13 (will get merge to the PR)
Thanks

Reply:
Modified